src/openms/include/OpenMS/KERNEL/MSSpectrum.h

```diff
@@ -251,6 +255,18 @@ namespace OpenMS
       retention_time_ = rt;
     }
 
+    /// Returns the absolute retention time (is seconds)
+    inline double getDriftTime() const
+    {
+      return drift_time_;
```

Comment:
we could add a special case here if `drift_time_ < 0` (it is seconds/milliseconds so it should never be less than zero) and then go into the precursors and search for drift times...